src/hooks/useFetch.ts

```diff
@@ -76,7 +76,7 @@ const useFetch = <Data = any, FetchParams extends any[] = any[]>(
     return () => {
       isReady.current = false;
     };
-  }, []);
+  });
```

Comment:
Removing the empty deps array will result in this hook running at each render. But we shouldn't call `getData()` at each render. I think we should find a way to leave it without complaints from the linter